common.mk

```diff
@@ -117,7 +121,13 @@ help: ## Display this help (default)
 ##@ Tooling Binaries
 ## --------------------------------------
 
-tools: $(TOOLING_BINARIES) ## Build tooling binaries
+check_tools:
+	# Check YTT Version or fail.
+	# Could do something more clever reusing versions in hack/tools/Makefile someday.
```

Comment:
The above two comments are printed to the screen.  If you want to suppress them you need to prefix even the comments with `@`

Reply:
(no specific urge to suppress) seems like its good debugging info ?  but good to note.